spec/models/rc_teacher_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe RcTeacher, type: :model do
+  context 'when validating presence' do
+    # it { is_expected.to validate_presence_of(:user_id) }
+    it { is_expected.to validate_presence_of(:review_center_id) }
```

Comment:
No need to validate the foreign key. You already have the test for the associations